packs/gatsby-build/config.yml

```diff
@@ -0,0 +1,29 @@
+
+# Name of the pack
+name: Gatsby Build
+
+# Description of the pack
+description: |+
+  The template pack allows you to get visibilility into the build time of your Gatsby Site,
+  using Open Telemetry to collect each step as Trace Data
```

Comment:
Is there a little bit more info we could provide? Like what are these steps?
`as Trace Data` -> `as spans in a distributed trace` ?

Reply:
Ye these should be as spans in a DT